mne/export/_export.py

```diff
@@ -2,10 +2,10 @@
 # License: BSD-3-Clause
 # Copyright the MNE-Python contributors.
 
-import os.path as op
+import os
 
-from ..utils import _check_fname, _validate_type, logger, verbose, warn
-from ._egimff import export_evokeds_mff
+from mne.export._egimff import export_evokeds_mff
+from mne.utils import _check_fname, _validate_type, logger, verbose, warn
```

Comment:
why the switch to absolute here? our convention is to use absolute imports only in tests. please revert (or explain why this is needed)

Reply:
ah, right you are, thanks for the reminder.